sdk/loader-instruction/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "solana-loader-instruction"
```

Comment:
nit: can we call this `solana-loader-v2-interface`? the `v2` bit so that people are less tempted to use it, especially when they see v3 and v4, and `interface` for consistency with the other loader crates, even though there's no state.

I could even be convinced to go with `v1` since they share an interface.